src/backend/base/langflow/base/io/chat.py

```diff
@@ -82,9 +82,9 @@ def build_with_record(
         if not return_message:
             message_text = message.text
         else:
-            message_text = message
+            message_text = message  # type: ignore
```

Comment:
Assuming we can update the return type of this method. Not sure in what case it would be an interator though, so maybe not worth digging into now 
```
    text: Optional[str | AsyncIterator | Iterator] = Field(default="")
```
`mesage.text type ^`